c_emulator/rv_cfg_func.h

```diff
@@ -0,0 +1,85 @@
+// vim: set tabstop=4 shiftwidth=4 expandtab
+// ============================================================================
+// Filename:    rv_cfg_func.h
+//
+// Description: Functions prototype support for rv_cfg
+//
+// Author(s):   Bill McSpadden (bill@riscv.org)
+//
+// Revision:    See git log
+// ============================================================================
+
+#pragma once
+
+#include "sail.h" 
+#include <libfyaml.h>
```

Comment:
libfyaml is an implementation detail so this include should move to the source file.